### PR TITLE
Added the The Official Raspberry Pi Handbook 2023[pdf]

### PR DESCRIPTION
**Pull request by ShivamPathak99**
## What does this PR do?
 | Improve repo 

## For resources
[The Official Raspberry Pi Handbook 2023 (204-page PDF)](https://magpi.raspberrypi.com/books/handbook-2023/pdf/download)
### Description
It is the most recent edition of official Raspberry Pi Handbook. It consists of collection of Inspirational projects which totally belongs to DIY category.

### Why is this valuable (or not)?
This handbook has been prepared by the makers of the MagPi that is the official Raspberry Pi magazine. Here the ideas has been conveyed in very creative way, the authors have used attractive photos  and interactive design for quickly dragging the readers attention.

### How do we know it's really free?
It is clearly mentioned on **[the official TheMagPi Website](https://magpi.raspberrypi.com/books/handbook-2023)** that the soft copy of the Handbook is freely available to download. 

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!
- **Please add the ***Hacktoberfest*** tag.**

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2079,8 +2079,10 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### Raspberry Pi
 
+* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:2082]:
```suggestion
* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed) - The MagPi magazine (PDF)
```

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2079,8 +2079,10 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
+* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:2085]:
```suggestion
* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023) - The MagPi magazine (PDF)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Completing with author/editor and PDF format notes

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**